src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/prisonersearch/PrisonerSearchClient.kt

```diff
@@ -55,8 +55,8 @@ enum class MatchedBy {
 
 data class PrisonerSearchRequest(
   val prisonNumber: String? = null,
-  val firstName: String,
-  val lastName: String?,
+  val firstName: String?,
```

Comment:
This is part of the fix to the request mapping problem - it turns out the API we are calling (match-prisoners) says that if you are searching by name, then you must provide at least a lastname